src/controllers/managerController.js

```diff
@@ -1,9 +1,11 @@
+/* eslint-disable import/no-cycle */
```

Comment:
Please investigate, rather than suppress the issue of cyclic import dependencies, it affects about 5 files